docs/DeveloperGuide.md

```diff
@@ -963,15 +1017,40 @@ testers are expected to do more *exploratory* testing.
 
     1. Prerequisites: List all persons using the `list` command.
 
-    1. Test case: `display 2`<br>
+    2. Test case: `display 2`<br>
         Expected: Second contact is displayed from the list.
 
-    1. Test case: `display 0`<br>
+    3. Test case: `display 0`<br>
        Expected: The default contact (i.e, first contact) is displayed. Error details shown in the status message.
 
-    1. Other incorrect delete commands to try: `display`, `display x` (where x is larger than the list size)<br>
+    4. Other incorrect delete commands to try: `display`, `display x` (where x is larger than the list size)<br>
        Expected: Similar to previous.
 
+### Editing a person
+
+1. Editing a contact while all contacts are being shown
+
+2. Editing the name of the contact
+  * Test case: `edit 1 n/bob`
+    Expected: The name of the first contact is edited to 'bob'
+
+3. Editing the age and interests of the contact
+  * Test case `edit 2 age/22 i/(1) shopping`
+    Expected: The age of the second contact as well as the first item in its interests list are edited
+
+4. Editing the interests list of a specific contact
+  * Test case `edit 4 i/(1) remove i/run`
```

Comment:
I think you can split up this test case into 3. So one to add and interest, another to edit an interest and another one to remove. Then u can have one more test case that does all 3 at once.